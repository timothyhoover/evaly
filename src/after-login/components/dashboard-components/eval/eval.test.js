import React from 'react';
import { render } from '@testing-library/react';
import Eval from './eval.component';

describe('Text is rendered correctly', () => {
	test('Successfully rendered title', () => {
		const component = render(<Eval />);
		expect(component).toMatchSnapshot();
	});
});
