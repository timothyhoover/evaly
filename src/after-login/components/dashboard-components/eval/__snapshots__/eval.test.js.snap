// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Text is rendered correctly Successfully rendered title 1`] = `
Object {
  "asFragment": [Function],
  "baseElement": <body>
    <div>
      <div
        class="eval"
      >
        <h1
          class="eval__title"
        >
          Let’s evaluate your current level
        </h1>
        <div
          class="eval__cards"
        >
          <div
            class="eval__card"
          >
            <svg
              class="icon eval__card-icon"
            >
              <use
                href="sprite.svg#book"
              />
            </svg>
            <div
              class="eval__card-content-reading"
            >
              <h3
                class="eval__card-title"
              >
                Reading
              </h3>
              <p
                class="eval__card-percentage"
              >
                25%
              </p>
            </div>
          </div>
          <div
            class="eval__card"
          >
            <svg
              class="icon eval__card-icon"
            >
              <use
                href="sprite.svg#pencil"
              />
            </svg>
            <div
              class="eval__card-content-writing"
            >
              <h3
                class="eval__card-title"
              >
                Writing
              </h3>
              <p
                class="eval__card-percentage"
              >
                25%
              </p>
            </div>
          </div>
          <div
            class="eval__card"
          >
            <svg
              class="icon eval__card-icon"
            >
              <use
                href="sprite.svg#ear"
              />
            </svg>
            <div
              class="eval__card-content-listening"
            >
              <h3
                class="eval__card-title"
              >
                Listening
              </h3>
              <p
                class="eval__card-percentage"
              >
                25%
              </p>
            </div>
          </div>
          <div
            class="eval__card"
          >
            <svg
              class="icon eval__card-icon"
            >
              <use
                href="sprite.svg#speakerphone"
              />
            </svg>
            <div
              class="eval__card-content-speaking"
            >
              <h3
                class="eval__card-title"
              >
                Speaking
              </h3>
              <p
                class="eval__card-percentage"
              >
                25%
              </p>
            </div>
          </div>
          <div
            class="eval__btn-box"
          >
            <button
              class="btn-secondary eval__button"
            >
              Start
            </button>
          </div>
        </div>
      </div>
    </div>
  </body>,
  "container": <div>
    <div
      class="eval"
    >
      <h1
        class="eval__title"
      >
        Let’s evaluate your current level
      </h1>
      <div
        class="eval__cards"
      >
        <div
          class="eval__card"
        >
          <svg
            class="icon eval__card-icon"
          >
            <use
              href="sprite.svg#book"
            />
          </svg>
          <div
            class="eval__card-content-reading"
          >
            <h3
              class="eval__card-title"
            >
              Reading
            </h3>
            <p
              class="eval__card-percentage"
            >
              25%
            </p>
          </div>
        </div>
        <div
          class="eval__card"
        >
          <svg
            class="icon eval__card-icon"
          >
            <use
              href="sprite.svg#pencil"
            />
          </svg>
          <div
            class="eval__card-content-writing"
          >
            <h3
              class="eval__card-title"
            >
              Writing
            </h3>
            <p
              class="eval__card-percentage"
            >
              25%
            </p>
          </div>
        </div>
        <div
          class="eval__card"
        >
          <svg
            class="icon eval__card-icon"
          >
            <use
              href="sprite.svg#ear"
            />
          </svg>
          <div
            class="eval__card-content-listening"
          >
            <h3
              class="eval__card-title"
            >
              Listening
            </h3>
            <p
              class="eval__card-percentage"
            >
              25%
            </p>
          </div>
        </div>
        <div
          class="eval__card"
        >
          <svg
            class="icon eval__card-icon"
          >
            <use
              href="sprite.svg#speakerphone"
            />
          </svg>
          <div
            class="eval__card-content-speaking"
          >
            <h3
              class="eval__card-title"
            >
              Speaking
            </h3>
            <p
              class="eval__card-percentage"
            >
              25%
            </p>
          </div>
        </div>
        <div
          class="eval__btn-box"
        >
          <button
            class="btn-secondary eval__button"
          >
            Start
          </button>
        </div>
      </div>
    </div>
  </div>,
  "debug": [Function],
  "findAllByAltText": [Function],
  "findAllByDisplayValue": [Function],
  "findAllByLabelText": [Function],
  "findAllByPlaceholderText": [Function],
  "findAllByRole": [Function],
  "findAllByTestId": [Function],
  "findAllByText": [Function],
  "findAllByTitle": [Function],
  "findByAltText": [Function],
  "findByDisplayValue": [Function],
  "findByLabelText": [Function],
  "findByPlaceholderText": [Function],
  "findByRole": [Function],
  "findByTestId": [Function],
  "findByText": [Function],
  "findByTitle": [Function],
  "getAllByAltText": [Function],
  "getAllByDisplayValue": [Function],
  "getAllByLabelText": [Function],
  "getAllByPlaceholderText": [Function],
  "getAllByRole": [Function],
  "getAllByTestId": [Function],
  "getAllByText": [Function],
  "getAllByTitle": [Function],
  "getByAltText": [Function],
  "getByDisplayValue": [Function],
  "getByLabelText": [Function],
  "getByPlaceholderText": [Function],
  "getByRole": [Function],
  "getByTestId": [Function],
  "getByText": [Function],
  "getByTitle": [Function],
  "queryAllByAltText": [Function],
  "queryAllByDisplayValue": [Function],
  "queryAllByLabelText": [Function],
  "queryAllByPlaceholderText": [Function],
  "queryAllByRole": [Function],
  "queryAllByTestId": [Function],
  "queryAllByText": [Function],
  "queryAllByTitle": [Function],
  "queryByAltText": [Function],
  "queryByDisplayValue": [Function],
  "queryByLabelText": [Function],
  "queryByPlaceholderText": [Function],
  "queryByRole": [Function],
  "queryByTestId": [Function],
  "queryByText": [Function],
  "queryByTitle": [Function],
  "rerender": [Function],
  "unmount": [Function],
}
`;
